lib/bundle/installer.rb

```diff
@@ -42,7 +43,7 @@ def install(entries, global: false, file: nil, no_lock: false, no_upgrade: false
           puts Formatter.success("#{verb} #{name}")
           true
         else
-          puts "Using #{name}"
+          puts "Using #{name}" unless quiet
```

Comment:
don't we need this in the positive case of this conditional as well? otherwise, we'll still get `Upgrading...`/`Tapping...`/`Installing...` in the output.

Reply:
I think it makes sense to output when something is being done and skip output when it's a no-op like this.

Can iterate on this post-merge.